### ISSUE TITLE
IPOPT Error when Using Drake from Python

### ISSUE THREAD

**Issue by bsbretly** [RobotLocomotion/drake#11894]
Hello,

I am following the instructions [here](https://drake.mit.edu/python_bindings.html#using-python-bindings) to install the python bindings for drake. 

I ran the `install_prereqs` script from my `/opt/drake/share/drake/setup/` directory successfully.

As stated in the instructions, I run the following commands:
```
git clone https://github.com/RobotLocomotion/drake.git
mkdir drake-build
cd drake-build
cmake ../drake
make -j
```
While running  `make -j` from my `~/drake-build` directory, I encounter the following error:
```
INFO: Call stack for the definition of repository 'ipopt' which is a pkg_config_repository (rule definition at /home/brett/drake-build/_bazel_brett/4db2c1afdf221ed1a5ae63ad68dbfb3d/external/drake/tools/workspace/pkg_config.bzl:262:25):
 - /home/brett/drake-build/_bazel_brett/4db2c1afdf221ed1a5ae63ad68dbfb3d/external/drake/tools/workspace/ipopt/repository.bzl:20:5
 - /home/brett/drake/tools/workspace/default.bzl:158:9
 - /home/brett/drake/WORKSPACE:10:1
ERROR: An error occurred during the fetch of repository 'ipopt':
   Unable to complete pkg-config setup for @ipopt repository: error 1 from [/usr/bin/pkg-config, "ipopt"]: 
```
I am running Ubuntu 16.04 with Python 2.7.

Thank you for your help!
Brett

**Comment by jwnimmer-tri** [comment]
Please post exactly the list of command(s) you are running that produce that error.

If you are compiling Drake, my first guess is that `setup/ubuntu/source_distribution/install_prereqs.sh` was forgotten, or did not complete successfully.

From the issue template:
> * Have you rerun `install_prereqs.sh` on your current commit of `drake`?

**Comment by bsbretly** [comment]
@jwnimmer-tri - thanks for the quick response. I have updated my original post with the information you requested.

**Comment by jwnimmer-tri** [comment]
Great, thanks!

Here's one problem: the `/opt/drake/share/drake/setup/install_prereqs` script is intended for users who have downloaded a pre-compiled Drake release and who want to just import and run the precompiled release.  It only installs the runtime depenencies, not the build-time dependencies.

To install build-time dependencies (i.e., to be able to to run CMake build of Drake), you'd need to run `setup/ubuntu/source_distribution/install_prereqs.sh` from the source tree.

**Comment by bsbretly** [comment]
@jwnimmer-tri - thank you! Running the script at `setup/ubuntu/source_distribution/install_prereqs.sh` was my problem - I no longer get the IPOPT issue.

I am, however, still unable to complete the build. Maybe it's easier to just include a screen grab of my terminal showing the error:
![image](https://user-images.githubusercontent.com/24208930/62426758-ddb3fa00-b6e0-11e9-90f1-efa7dff7c781.png)

As you can see, I tried to build again - still with no success. If it's more appropriate to make a new issue for this problem, I can happily do that as well. 

Thanks again!

**Comment by jwnimmer-tri** [comment]
Per https://drake.mit.edu/getting_help.html#asking-your-question we'd usually do this help on StackOverflow, but we can work it here also for now.

_"Internal compiler error: Killed"_ is usually an out-of-memory condition due to too-high parallelism.

To reduce parallelism, create a file `$HOME/.bazelrc` that contains a line like `build --jobs=2` or similar, to limit the number of CPUs used in parallel (so as to reduce memory pressure).  I would start with a value around 80% of your total vCPUs and work your way down until it succeeds.

**Comment by bsbretly** [comment]
Many thanks @jwnimmer-tri - your super quick feedback and patience is much appreciated. Everything has built - I will close this issue.
